File: src/main/java/tech/zolhungaj/amqapi/servercommands/gameroom/game/WaitingForBuffering.java
package tech.zolhungaj.amqapi.servercommands.gameroom.game;

import com.squareup.moshi.Json;
import tech.zolhungaj.amqapi.servercommands.CommandType;

@CommandType("quiz waiting buffering")
public record WaitingForBuffering(
        @Json(name = "firstSong")
        boolean isFirstSong
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/gameroom/game/WaitingForBuffering.kt
package tech.zolhungaj.amqapi.servercommands.gameroom.game

import com.squareup.moshi.Json
import tech.zolhungaj.amqapi.servercommands.CommandType

@JvmRecord
@CommandType("quiz waiting buffering")
data class WaitingForBuffering(
    @Json(name = "firstSong")
    val isFirstSong: Boolean
)
